tests/helpers/tests/applications/data.py

```diff
@@ -47,3 +49,17 @@ def get_checksum(name: str) -> str:
     paths to custom files.
     """
     return _pooch.registry[name]
+
+
+@pytest.fixture
+def ymir():
+    with open(get_path('ymir_detectors.json')) as f:
+        data = json.load(f)
+    return data
+
+
+@pytest.fixture
+def loki(large_file_test: bool):
+    assert large_file_test
+    with open(get_path('loki.json')) as f:
+        return json.load(f)
```

Comment:
I think they should go to `conftest.py` instead.
This module is also used by documentations.